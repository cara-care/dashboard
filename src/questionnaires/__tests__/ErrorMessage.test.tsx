import React from 'react';
import { render } from '@testing-library/react';
import ErrorMessage from '../components/ErrorMessage';

describe('<ErrorMessage />', () => {
  it('renders correctly', () => {
    const { container } = render(<ErrorMessage message="Test Error" />);
    expect(container).toMatchSnapshot();
  });
});
