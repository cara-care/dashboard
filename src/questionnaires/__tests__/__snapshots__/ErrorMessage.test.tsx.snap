// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> renders correctly 1`] = `
<div>
  <div
    class="makeStyles-content-1"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-colorError"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
      />
    </svg>
    <p
      class="MuiTypography-root MuiTypography-body2"
    >
      <span>
        Test Error
      </span>
    </p>
  </div>
</div>
`;
